### ISSUE TITLE
[a11y] - Labs (staging)

### ISSUE THREAD

**Issue by trallard** [Quansight/Quansight-website#156]
- 🔗  [Reference document](https://docs.google.com/document/d/1B3Wr17-1Ic1X6uJe7UQQTzBnfB09rPTatEsax82jIcs/edit)
- [Preview (staging) for LLC](https://github.com/Quansight/Quansight-website/deployments/activity_log?environment=Preview+%E2%80%93+quansight-labs)
- [Production for LLC](https://github.com/Quansight/Quansight-website/deployments/activity_log?environment=Production+%E2%80%93+quansight-consulting](https://github.com/Quansight/Quansight-website/deployments/activity_log?environment=Production+%E2%80%93+quansight-labs))
- 📖  [Gov uk a11y checklist](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag )

---

## Criteria

- [ ] Site is keyboard accessible
	- [ ] Tab order is logical
	- [ ] Focus is never trapped in a loop

- [ ] Forms inputs have explicit labels 
	- [ ] Forms can be filled with just the keyboard
	- [ ] Forms contain title attributes AND/OR
	- [ ] Forms contain legends, labels, hin text, and tooltips
	- [ ] Form instructions are associated with inputs

- [ ] Provide alternative content for non-text content
	- [ ] All images have an alt attribute

- [ ] Text has sufficient color contrast
	- [ ] All text has a contrast ratio of 4.5:1 with the background
	- [ ] Links can be easily distinguished from surrounding text

- [ ]  Headers are nested properly
	- [ ] Only one H1 per page
	- [ ] Headings are nested in a logical way
	- [ ] Subsection hierarchy is preserved

- [ ] Language is set
	- [ ] The language attribute is set in all pages (in this case English throughout)
	
- [ ] Site can be used without stylesheets
	- [ ] The site does not rely on CSS to be functional or understandable
	
- [ ] Links are unique and descriptive
	- [ ] All links can be understood taken alone, e.g., ‘Read more about 508’ OR
	- [ ] Links provide an adequate title or aria attribute
	- [ ] Links that open in a new window indicate so through an aria-label or title

- [ ] Page titles are descriptive
	- [ ] Page titles are in plain English
	- [ ] Titles describe the site and page being displayed

**Comment by gabalafou** [comment]
As discussed with @trallard, there's no need to have two separate issues for production versus staging for accessibility auditing. They can be done on staging or production, the result should be the same. So I'm closing all of the versions of this issue marked for staging.
